cfme/base/ui.py

```diff
@@ -766,6 +766,7 @@ class RegionView(ConfigurationView):
     @View.nested
     class details(Tab):  # noqa
         TAB_NAME = "Details"
+        table = Table(locator="//td[@title='Edit this Region']/../../..")
```

Comment:
REQUIRED: I'd rather change this to something like Table(locator=".//table[./tbody/tr/td[blabla]]")

Reply:
fixed